app/models/track.rb

```diff
@@ -1,3 +1,4 @@
 class Track < ActiveRecord::Base
+  has_and_belongs_to_many :programs
```

Comment:
Declining to change because the link between programs and tracks is not rich; there is nothing to be gained by has_many :through.